examples/miral-shell/miral-xrun.sh

```diff
@@ -42,7 +42,10 @@ fi
 
 if [ "${x11_server}" == "Xmir" ];
 then
-  if   [ -e "${XDG_RUNTIME_DIR}/miral_socket" ];
+  if [ -v MIR_SOCKET ]
```

Comment:
Huh, I can't find `-v` documented anywhere... Any pointers? I'm assuming this tests for the variable to be defined, but what if it's empty? Wouldn't `[ -n "${MIR_SOCKET}" ]` be better here?

Reply:
$ help test | grep \\\-v
      -v VAR         True if the shell variable VAR is set.

If it's empty, that's (apparently) what is intended.